Makefile

```diff
@@ -120,6 +119,27 @@ docker-push:
 	@echo ${DOCKERHUB_TOKEN} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
 	docker push ${IMG_W_TAG}
 
-help:
-	@grep -E '^[a-zA-Z_-]+:.*$$' $(MAKEFILE_LIST) | sort
+# Targets for release prep.
+create-local-tag-for-major-release:
+	${MAKEFILE_PATH}/scripts/create-local-tag-for-release -m
+
+create-local-tag-for-minor-release:
+	${MAKEFILE_PATH}/scripts/create-local-tag-for-release -i
+
+create-local-tag-for-patch-release:
+	${MAKEFILE_PATH}/scripts/create-local-tag-for-release -p
+
+update-versions-for-release:
+	${MAKEFILE_PATH}/scripts/update-versions-for-release
+
+release-prep-major: create-local-tag-for-major-release update-versions-for-release
+
+release-prep-minor: create-local-tag-for-minor-release update-versions-for-release
+
+release-prep-patch: create-local-tag-for-patch-release update-versions-for-release
+
+release-prep-custom: # Run make NEW_VERSION=1.2.3 release-prep-custom to prep for a custom release version
```

Comment:
Hm.. I'm still of a mind that this is the only `release-prep` target we really need. Whoever is prepping the release needs to do the manual work of figuring out if it's a major, minor, or patch release anyway which means they'll know what the version should be before executing any of these targets. Or maybe I'm missing some other use case??

Reply:
release-prep-custom target is useful for custom versions i.e. not the usual major / minor/ patch increment. 
It requires the user to know the latest tag. 

Since this is a script to automate the whole process of creation of a tag, release-prep-{major/minor/patch} is an effort to cover 90+% of the cases.

Reply:
That makes sense.

> It requires the user to know the latest tag.

Right, I was thinking from the prospective of needing to cut a release after multiple commits and the person doing the release needs to review all the changes anyway to determine what kind of release should be cut.

I think with the more-frequent-releases approach, being able to run `release-prep-patch/minor/major` with just my changes + version upgrades becomes more useful. I'm onboard👍🏼